tidb-cloud/serverless-limitations.md

```diff
@@ -8,49 +8,51 @@ aliases: ['/tidbcloud/serverless-tier-limitations']
 
 ## Limitations
 
-### SQL
+### Audit Logs
 
-- [Time to live (TTL)](/time-to-live.md) is not available for TiDB Serverless clusters currently.
-- The [`FLASHBACK CLUSTER TO TIMESTAMP`](/sql-statements/sql-statement-flashback-to-timestamp.md) syntax is not applicable to [TiDB Serverless](/tidb-cloud/select-cluster-tier.md#tidb-serverless-beta) clusters.
-- The [`SLEEP()` function](/functions-and-operators/miscellaneous-functions.md) only supports a maximum sleep time of 300 seconds.
+- Database audit logging is currently unavailable.
+- Console audit logging is currently unavailable.
 
-### System tables
+### Connection
 
-- Tables `CLUSTER_SLOW_QUERY`, `SLOW_QUERY`, `CLUSTER_STATEMENTS_SUMMARY`, `CLUSTER_STATEMENTS_SUMMARY_HISTORY`, `STATEMENTS_SUMMARY`, `STATEMENTS_SUMMARY_HISTORY` are not available for TiDB Serverless clusters.
+- Only [Public Endpoint](/tidb-cloud/connect-via-standard-connection-serverless.md) and [Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections-serverless.md) can be used. You cannot use [VPC Peering](/tidb-cloud/set-up-vpc-peering-connections.md) to connect to TiDB Serverless clusters. 
+- No "IP Access List" support.
 
-### Transaction
+### Encryption
 
-- The total size of a single transaction is set to no more than 10 MB on TiDB Serverless during the beta phase.
+- Data persisted in your TiDB Serverless cluster undergoes encryption utilizing the encryption tool provided by the cloud provider responsible for managing your cluster. However, TiDB Serverless does not provide additional optional measures for protecting data at-rest on disks beyond infrastructure-level encryption.
+- Using [customer-managed encryption keys (CMEK)](/tidb-cloud/tidb-cloud-encrypt-cmek.md) is currently unavailable.
 
-### Connection
+### Maintenance window
 
-- Only [Standard Connection](/tidb-cloud/connect-via-standard-connection-serverless.md) and [Private Endpoint](/tidb-cloud/set-up-private-endpoint-connections-serverless.md) can be used. You cannot use [VPC Peering](/tidb-cloud/set-up-vpc-peering-connections.md) to connect to TiDB Serverless clusters.
-- No "IP Access List" support.
+- [Maintenance window](/tidb-cloud/configure-maintenance-window.md) is currently unavailable.
 
-### Monitoring
+### Monitoring and Diagnosis
 
-- [Third-party Monitoring integrations](/tidb-cloud/third-party-monitoring-integrations.md) are currently not available for TiDB Serverless.
-- [Cluster Events](/tidb-cloud/tidb-cloud-events.md) are currently not available for TiDB Serverless.
-- [Built-in Alerting](/tidb-cloud/monitor-built-in-alerting.md) is currently not available for TiDB Serverless.
+- [Third-party Monitoring integrations](/tidb-cloud/third-party-monitoring-integrations.md) are currently unavailable.
+- [Built-in Alerting](/tidb-cloud/monitor-built-in-alerting.md) is currently unavailable.
+- [Key Visualizer](/tidb-cloud/tune-performance.md#key-visualizer) is currently unavailable.
```

Comment:
Is [Index Insight](https://docs.pingcap.com/tidbcloud/tune-performance#index-insight-beta) available for Serverless? Its doc says **only available for Dedicated**.

Reply:
Added
In addition,  [Cluster Events](https://docs.pingcap.com/tidbcloud/tidb-cloud-events) is unavailable as well.